modules/microshift-nw-multus-add-pod.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microhift-cni-multus.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-multus-add-pod_{context}"]
+= Adding a pod to an additional network
+
+You can add a pod to an additional network. The pod continues to send normal cluster-related network traffic over the default network.
+
+When a pod is created additional networks are attached to it. However, if a pod already exists, you cannot attach additional networks to it.
```

Comment:
Maybe we can say: "if you want to attach additional network to an existing Pod, you need to restart the Pod" or something like this